superset/assets/backendSync.json

```diff
@@ -1683,18 +1683,6 @@
       "renderTrigger": true,
       "default": ""
     },
-    "where": {
```

Comment:
I haven't touched this file before- what is it doing exactly?

Reply:
That represents the old TextControl where people would write their WHERE clause. Did we even run a db migration to convert the old filters (where & having) to new ones?

Reply:
Oh I see it's in this PR !

Reply:
@mistercrunch I'm hoping to clean up the backend later this week so we can remove all references to the old where, having clauses entirely.